src/EnergyPlus/CMakeLists.txt

```diff
@@ -880,8 +879,28 @@ install(TARGETS energyplus energyplusapi DESTINATION ./)
 
 if(LINK_WITH_PYTHON)
   # link the executable to the dll and bring it in
-  get_filename_component(RESOLVED_PYTHON_LIBRARY "${PYTHON_LIBRARIES}" REALPATH)
+  get_filename_component(RESOLVED_PYTHON_LIBRARY "${Python_LIBRARIES}" REALPATH)
   if(WIN32)
+
+    # In case you have both release and debug Python libraries on your system, Python_LIBRARIES might be "optimized;C:/.../python38.lib;debug;C:/.../python38_d.lib"
+    # so it can't be used directly, we could use a generator expression to find the $<TARGET_FILE:Python::Python> library used and that'd point directly
+    # to the DLL. But it won't work nicely for the install(code ...) command below... so we do a hack, get the optimized one always...
+    list(LENGTH Python_LIBRARIES _LEN)
+    if (_LEN GREATER 1)
+      set (_Python_DOING_RELEASE FALSE)
+      foreach(_currentArg ${Python_LIBRARIES})
+        message("_currentArg=${_currentArg}")
+        if ("x${_currentArg}" STREQUAL "xoptimized")
+          set(_Python_DOING_RELEASE TRUE)
+        elseif(_Python_DOING_RELEASE)
+          get_filename_component(RESOLVED_PYTHON_LIBRARY "${_currentArg}" REALPATH)
+          break()
+        endif()
+      endforeach()
+    # else()
+      # No-op, already done above
+    endif()
```

Comment:
Actual fix for #8778 when you have both Release and Debug python libs on your system.